core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiService.scala

```diff
@@ -87,6 +87,22 @@ trait AdminApiService extends HttpService with PerRequestCreator with UserInfoDi
           WorkspaceService.AdminAbortSubmission(WorkspaceName(workspaceNamespace, workspaceName), submissionId))
       }
     } ~
+    path("admin" / "realms") {
+      post { //create a realm
```

Comment:
Do we prefer `PUT` or `POST` for create services? I took a quick look at some other services and it currently looks inconsistent.